test-unit/lobster-codebeamer/test_codebeamer_schema.py

```diff
@@ -0,0 +1,83 @@
+import unittest
+from lobster.items import Tracing_Tag, Requirement, Implementation, Activity
+from lobster.location import Codebeamer_Reference
+from lobster.errors import LOBSTER_Error
+from lobster.tools.codebeamer.codebeamer import _create_common_params, _create_lobster_item
+ 
+class TestCreateFunctions(unittest.TestCase):
+
+    def setUp(self):
+        self.root_url = 'http://root_url'
+        self.cb_item_template = {
+            'version': 1,
+            'tracker': {'id': 123}
+        }
+ 
+    def generate_cb_item(self, item_id, name):
+        """Generate a codebeamer item dictionary."""
+        return {
+            'id': item_id,
+            **self.cb_item_template,
+            'name': name
+        }
+ 
+    def generate_common_params(self, namespace, item_name, kind, expected_class):
+        """Generate a test case for common params and lobster item creation."""
+        cb_item = self.generate_cb_item(1, item_name) 
+        common_params = _create_common_params(namespace, cb_item, self.root_url, item_name, kind)
+       
+        return {
+            'common_params': common_params,
+            'item_name': item_name,
+            'expected_class': expected_class,
+            'tag': Tracing_Tag(namespace, str(cb_item["id"]), cb_item["version"]),
+            'location': Codebeamer_Reference(self.root_url, cb_item["tracker"]["id"], cb_item["id"], cb_item["version"], item_name),
+            'kind' : kind
+        }
+
+    def  generate_test_case(self):
+
```

Comment:
No need to add a blank line at the start of the function. Please check for other functions and classes as well.